DateOnlyTimeOnly.AspNet.Swashbuckle/DateOnlyTimeOnly.AspNet.Swashbuckle.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="6.2.3" />
+    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerGen" Version="6.3.0" />
```

Comment:
Why force users to update `SwaggerGen`? What if they want to stay on a lower version (`6.2.3` is the first version that targets `net6.0`).

Reply:
For me it makes sense to upgrade to the current version but that is of course not a must.

Reply:
I'm saying that this a dependency of a NuGet package, and users that have lower `SwaggerGen` version simply won't be able to use it after this change.
You can still use latest version of `SwaggerGen` in your project. `PackageReference` above only specifies a minimum required version.